core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1139,7 +1139,7 @@ object KafkaConfig {
       .define(MetadataMaxRetentionMillisProp, LONG, Defaults.LogRetentionHours * 60 * 60 * 1000L, null, HIGH, MetadataMaxRetentionMillisDoc)
 
       /************* Authorizer Configuration ***********/
-      .define(AuthorizerClassNameProp, STRING, Defaults.AuthorizerClassName, LOW, AuthorizerClassNameDoc)
+      .define(AuthorizerClassNameProp, STRING, Defaults.AuthorizerClassName, new ConfigDef.NonNullValidator(), LOW, AuthorizerClassNameDoc)
```

Comment:
I think Scala allows you to just write `ConfigDef.NonNullValidator` without the `new` keyword and the `()`, but I might be wrong.

Reply:
This should be fine, I see the same in this file : `new RaftConfig.ControllerQuorumVotersValidator()`

Reply:
Okay, sounds sensible.

Reply:
I suspect most people use a properties file to configure their brokers and in that case I'm not sure if a `STRING` configuration can ever be `null`.